components/brave_wallet_ui/stories/wallet-components.tsx

```diff
@@ -296,4 +299,30 @@ export const _Tabs = () => {
       onSelect={(option) => console.log(option)}
     />
   )
+}
+
+export const _MarketGrid = () => {
+  return (
+    <div
+      style={{
+        width: '700px'
+      }}
+    >
+      <MarketGrid
+        headers={marketGridHeaders}
+        coinMarketData={coinMarketMockData}
+        showEmptyState={false}
+        sortedBy='marketCap'
+        onSort={(columnId, sortOrder) => console.log(`sort by ${columnId} ${sortOrder}`)}
+        onSelectCoinMarket={() => {}}
+        isBuySupported={() => true}
+        isDepositSupported={() => false} 
+        onClickBuy={() => {}}
+        onClickDeposit={() => {}}
+        onUpdateIframeHeight={() => {}}
+        fiatCurrency={'USD'}
+      />
+
+    </div>
+  )
```

Comment:
Missing end of page line

Reply:
Adde new line...highlight here is wrong 😄